.github/workflows/manifest.yml

```diff
@@ -26,7 +26,7 @@ jobs:
           west init -l . || true
 
       - name: Manifest
-        uses: zephyrproject-rtos/action-manifest@v1.3.1
+        uses: zephyrproject-rtos/action-manifest@pull/14/head
```

Comment:
you can probably push a `v1` tag that points at the lastes `v1.x.x` and the updated it when needed, then you won't have to update it here, I think that's how actions are normally managed. This is fine as well though, you have to create this anyway for testing.

Reply:
ah yes, that is a very good idea! Will do this next time.